include/circt/Dialect/SV/TypeDecl.td

```diff
@@ -222,6 +218,33 @@ def GetModportOp: SVOp<"modport.get", [NoSideEffect]> {
   ];
 }
 
+def AssignInterfaceSignalOp : SVOp<"interface.signal.assign", []> {
+  let summary = "Assign an interfaces signal to some other signal.";
+
+  let arguments = (ins InterfaceType:$iface, FlatSymbolRefAttr:$signalName,
+    AnyType:$rhs);
+
+  let assemblyFormat = [{
+    $iface `->` $signalName `=` $rhs attr-dict `:` type($iface) `,` type($rhs)
+  }];
+}
+
+def ReadInterfaceSignalOp : SVOp<"interface.signal.read", [NoSideEffect]> {
+  let summary = "Access the data in an interface's signal.";
+
+  let arguments = (ins InterfaceType:$iface, FlatSymbolRefAttr:$signalName);
+  let results = (outs AnyType:$signalData);
+
+  let assemblyFormat = [{
+    $iface `->` $signalName attr-dict `:`
+      type($iface) `,` type($signalData)
+  }];
+
+  let builders = [
+    OpBuilderDAG<(ins "::mlir::Value":$iface, "::llvm::StringRef":$signalName)>
+  ];
+}
```

Comment:
Maybe add verifiers like in GetModport to check that the signal name exists in the interface.